commit 2471d5a1a7c8e3367ae9472dfbc0867377a72a1f
Author: Volker Krause <vkrause@kde.org>
Date:   Sat Jan 2 18:07:01 2021 +0100

    Consider pkgconfig sysroot for pkgdatadir
    
    For libs/cflags this is done automatically, but not for manually accessed
    variables. This matches what wayland-protocols does.

diff --git a/src/wayland-client.pc.in b/src/wayland-client.pc.in
index eef61da..8221a87 100644
--- a/src/wayland-client.pc.in
+++ b/src/wayland-client.pc.in
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
-pkgdatadir=@datadir@/@PACKAGE@
+pkgdatadir=${pc_sysrootdir}@datadir@/@PACKAGE@
 libdir=@libdir@
 includedir=@includedir@
 
diff --git a/src/wayland-scanner.pc.in b/src/wayland-scanner.pc.in
index 7b2a4c9..af50511 100644
--- a/src/wayland-scanner.pc.in
+++ b/src/wayland-scanner.pc.in
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
-pkgdatadir=@datadir@/@PACKAGE@
+pkgdatadir=${pc_sysrootdir}@datadir@/@PACKAGE@
 wayland_scanner=@bindir@/wayland-scanner
 
 Name: Wayland Scanner
diff --git a/src/wayland-server.pc.in b/src/wayland-server.pc.in
index 50dff53..a5fe9d7 100644
--- a/src/wayland-server.pc.in
+++ b/src/wayland-server.pc.in
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
-pkgdatadir=@datadir@/@PACKAGE@
+pkgdatadir=${pc_sysrootdir}@datadir@/@PACKAGE@
 libdir=@libdir@
 includedir=@includedir@
 
